packages/editor-ui/src/stores/ui.store.ts

```diff
@@ -628,6 +630,40 @@ export const useUIStore = defineStore(STORES.UI, () => {
 		lastCancelledConnectionPosition.value = undefined;
 	}
 
+	const isInstanceOwnerInCloud = () => {
```

Comment:
would this be a better fit in the cloud plans or users store? 
because ui store is already too big and general, and is slowly getting broken apart.